shared/chat/conversation/messages/popup.desktop.js

```diff
@@ -57,9 +58,15 @@ export const TextPopupMenu = ({message, onEditMessage, onDeleteMessage, onHidden
   let items = []
   if (message.author === you) {
     items = [
-      {title: 'Edit', onClick: () => onEditMessage(message)},
-      {title: 'Delete', subTitle: 'Deletes for everyone', danger: true, onClick: () => onDeleteMessage(message)},
+      {onClick: () => onEditMessage(message), title: 'Edit'},
+      {danger: true, onClick: () => onDeleteMessage(message), subTitle: 'Deletes for everyone', title: 'Delete'},
     ]
+
+    if (!flags.chatAdminOnly) {
+      // remote edit
+      items.shift()
```

Comment:
🐑 Would be nicer to have this next to where Edit is placed, so if the list above is changed it's obvious how this affects it. i.e. doing
```
...(flags.chatAdminOnly ? {onClick: ... title: 'Edit'} : {})
```